app/src/main/java/org/mozilla/fenix/components/Components.kt

```diff
@@ -196,6 +200,15 @@ class Components(private val context: Context) {
     val appStartReasonProvider by lazyMonitored { AppStartReasonProvider() }
     val startupActivityLog by lazyMonitored { StartupActivityLog() }
     val startupStateProvider by lazyMonitored { StartupStateProvider(startupActivityLog, appStartReasonProvider) }
+
+    val messagingStorage by lazyMonitored {
```

Comment:
nit: could we move this to the `Analytics` component since we have experiments there too.

Reply:
👍🏽